voice.js

```diff
@@ -0,0 +1,183 @@
+let personName = "";
+let toDoList = [];
+
+function greetWithName(commandArray) {
+  const userName = commandArray.slice(-1);
+  if (personName) {
+    console.log(`You already mention your name is : ${userName}`);
+    return;
+  } else {
+    personName = userName;
+    return `Nice to meet you,${userName}.`;
```

Comment:
It would be good to be consistent about answering our user with a `return` or a `console.log`. I recommend using `return` since you are demoing the outcome of your code with `console.log` at the end of the file.
I'll let you make the changes to the other functions.